cms/djangoapps/contentstore/management/commands/swap_from_auto_track_cohort_pilot.py

```diff
@@ -0,0 +1,192 @@
+from contentstore.course_group_config import GroupConfiguration
```

Comment:
Please move this migration command and admin table into openedx/core/djangoapps/verified_track_content, so it lives with the code for the old cohorting feature that we are migrating from (ultimately we will want to delete all of this code).